qcelemental/models/molecule.py

```diff
@@ -768,7 +768,6 @@ def get_hash(self):
         m = hashlib.sha1()
         concat = ""
 
-        np.set_printoptions(precision=16)
```

Comment:
Would the below do what you need or is the precision=16 actually throwing off your hashes in some way?

```
with np.printoptions(precision=16):
    for field
       ...
       concat +=
```

Reply:
From my testing this actually does not change any hashes. It just is surprising to have `mol.get_hash()` change the output of printed arrays.